packages/dx-react-grid-material-ui/package.json

```diff
@@ -86,7 +87,8 @@
     "@devexpress/dx-react-core": "1.0.0-alpha.2",
     "@devexpress/dx-react-grid": "1.0.0-alpha.2",
     "prop-types": "^15.5.8",
-    "react": "^15.5.4"
+    "react": "^15.5.4",
+    "react-tap-event-plugin": "^2.0.1"
```

Comment:
react-tap-event-plugin is required by the material-ui library and should be referenced as peer dependency.